explorer/src/components/Staking/OperatorsList.tsx

```diff
@@ -189,6 +189,50 @@ export const OperatorsList: FC = () => {
           <div>{`${bigNumberToNumber(row.original.currentTotalStake)} ${selectedChain.token.symbol}`}</div>
         ),
       },
+    ]
+    if (useRpcData && deposits.find((d) => d.account === subspaceAccount))
+      cols.push({
+        accessorKey: 'deposits',
+        header: 'Deposits',
+        enableSorting: !useRpcData,
+        cell: ({
+          row,
+        }: Cell<
+          OperatorsConnectionQuery['operatorsConnection']['edges'][0]['node'] | Operators
+        >) => {
+          const opDeposits = deposits.filter((d) => d.operatorId.toString() === row.original.id)
+          const depositShares = opDeposits.reduce(
+            (acc, deposit) => acc + BigInt(deposit.shares),
+            BigInt(0),
+          )
```

Comment:
In local testing I am getting some `deposit.shares` values in scientific notation which creates a failure here.